sdks/java/io/cdc/src/main/java/org/apache/beam/io/cdc/DebeziumHistory.java

```diff
@@ -0,0 +1,75 @@
+package org.apache.beam.io.cdc;
+
+import io.debezium.document.Document;
+import io.debezium.document.DocumentReader;
+import io.debezium.document.DocumentWriter;
+import io.debezium.relational.history.AbstractDatabaseHistory;
+import io.debezium.relational.history.DatabaseHistoryException;
+import io.debezium.relational.history.HistoryRecord;
+import org.apache.beam.sdk.transforms.splittabledofn.RestrictionTracker;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+
+public class DebeziumHistory extends AbstractDatabaseHistory {
```

Comment:
There's a `DebeziumSDFDatabaseHistory` file, can you remove it and apply the update in the code to use this new `DebeziumHistory`?

Also add the apache license at the first line of all your files

Reply:
I added the license and updated the code. 👨‍💻